extensions/amp-auto-ads/0.1/placement.js

```diff
@@ -199,13 +199,17 @@ export class Placement {
           this.state_ = PlacementState.TOO_NEAR_EXISTING_AD;
```

Comment:
Can you please update the unit test for these changes? The test file is here: extensions/amp-auto-ads/0.1/test/test-placement.js.

Reply:
Added.